src/App.vue

```diff
@@ -6,22 +6,24 @@
     <image-viewer ref="imageViewer" />
     <helper-window ref="helper" />
     <alert-window ref="alertWindow" />
+    <gdpr-modal />
```

Comment:
Is it on purpose ?

Reply:
no, I just corrected it. Forgot I had mixed it with some very preliminary stuff for GDPR. Sorry.